packages/algorand-js/src/interpreter/opcode.ts

```diff
@@ -22,6 +22,12 @@ export class Op {
     }
   }
 
+  assertArrLength (arr: Uint8Array[]): void {
+    if (arr.length > MAX_UINT8 + 1) {
+      throw new TealError(ERRORS.TEAL.ASSERT_ARR_LENGTH);
+    }
```

Comment:
it must be also bigger than 0 (we need to have at least one element)

Reply:
ok